TimerCalculatorUtility/TimerCalculatorUtility/ViewModels/TimerViewModel.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Diagnostics;
+using TimerCalculatorUtility.Enums;
+using TimerCalculatorUtility.Models;
+using Xamarin.Forms;
+
+namespace TimerCalculatorUtility.ViewModels
+{
+    public class TimerViewModel : INotifyPropertyChanged
+    {
+        private int _lapCount;
+        private Stopwatch _stopwatchLap = new Stopwatch();
+        private Stopwatch _stopwatchOverall = new Stopwatch();
+
+        public TimerViewModel()
+        {
+            Status = StopwatchStatus.Onstart;
+            StopCommand = new Command(Stop);
+            LapCommand = new Command(StartLap);
+            ResetCommand = new Command(Reset);
+            ResumeCommand = new Command(StartTimeRunning);
+            StartCommand = new Command(StartTimeRunning);
+        }
+        public string TimeString { get; set; } = "00:00.00";
+        public string LapTimeString { get; set; } = "00:00.00";
+        public string Text { get; set; }
+        public ObservableCollection<TimerItem> Times { get; set; } = new ObservableCollection<TimerItem>();
+        public Command StopCommand { get; set; }
```

Comment:
Assigning commands done only in constructor. There's no need to use setter for them.